it/src/test/java/com/expediagroup/streamplatform/streamregistry/it/DomainTestStage.java

```diff
@@ -95,6 +95,11 @@ public void upsert() {
     assertThat(specificationPart.getConfiguration().get(factory.key).asText(), is(factory.value));
   }
 
+  @Override
+  public void delete() {
+    //not implemented for domain
```

Comment:
Why? Is this not part of STREAMP-3135 along with zone and infra deletion?

Reply:
Elliot said it was just "nice to have" but not critical. https://confluence.expedia.biz/pages/viewpage.action?spaceKey=~ewest&title=Control+Plane+deletion+-+how+to+cascade+deletes

Reply:
Ah cool, thanks for the doc. It's good to have a reference. Will this be covered in a different ticket or a future PR? Whenever I hear "not critical", I think "might not actually happen" :D